tests/automation_framework/tests/work_order_tests/test_submit.py

```diff
@@ -1815,3 +1815,443 @@ def test_work_order_submit_requesterNonce_param_empty(self):
                 is TestStep.SUCCESS.value)
         logger.info('\t\t!!! Test completed !!!\n\n')
 
+    @pytest.mark.work_order_submit
+    @pytest.mark.test_workordersubmit_requestersignature_difflength
+    @pytest.mark.listener
+    def test_workordersubmit_requestersignature_difflength(self):
```

Comment:
Names for all test names should follow the new naming convention we discussed last week.